freeipa/src/main/java/com/sequenceiq/freeipa/controller/UserV1Controller.java

```diff
@@ -42,13 +42,16 @@
     public SynchronizeUserResponse synchronizeUser(SynchronizeUserRequest request) {
         String userCrn = checkUserCrn();
         LOGGER.debug("synchronizeUser() requested for user {}", userCrn);
+        String accountId = crnService.getCurrentAccountId();
 
-        return userService.synchronizeUser(userCrn);
+        return userService.synchronizeUser(accountId, userCrn, userCrn);
```

Comment:
@handavid - nit - userCrn is repeating here.

Reply:
this is on purpose. the userCrn is passed in first as the actor and secondly as the subject.

Reply:
ok sure, please add comments for the same in the code in later PRs.